Packs/AzureStorage/Integrations/AzureStorage/AzureStorage.py

```diff
@@ -0,0 +1,291 @@
+        if is_hns_enabled is not None:
+            json_data_args['properties']['isHnsEnabled'] = is_hns_enabled
+        if large_file_shares_state is not None:
+            json_data_args['properties']['largeFileSharesState'] = large_file_shares_state
+        if allow_blob_public_access is not None:
+            json_data_args['properties']['allowBlobPublicAccess'] = allow_blob_public_access
+        if minimum_tls_version is not None:
+            json_data_args['properties']['minimumTlsVersion'] = minimum_tls_version
+        print(json_data_args)
+        return self.ms_client.http_request(
+            method='PUT',
+            url_suffix=f'/resourceGroups/{self.resource_group_name}/providers/Microsoft.Storage/storageAccounts/'
+                       f'/{account_name}',
+            params={
+                'api-version': API_VERSION,
+            },
+            json_data=json_data_args,
+            timeout=30,
+        )
+
+
+def storage_account_list(client: AKSClient, args: Dict) -> CommandResults:
+    account_name = args.get('account_name', '')
+    response = client.storage_account_list_request(account_name)
+    accounts = response.get('value', [response])
+    readable_output = [{
+        'Account Name': account.get('name'),
+        'Subscription ID': re.search('subscriptions/(.+?)/resourceGroups', account.get('id')).group(1),
+        'Resource Group': re.search('resourceGroups/(.+?)/providers', account.get('id')).group(1),
+        'Kind': account.get('kind'),
+        'Sku': account.get('sku', {}),
+        'Status Primary': account.get('properties', {}).get('statusOfPrimary'),
+        'Status Secondary': account.get('properties', {}).get('statusOfSecondary'),
+        'Location': account.get('location'),
+        'Tags': account.get('tags')
+    } for account in accounts]
+    return CommandResults(
+        outputs_prefix='AzureStorage.StorageAccount',
+        outputs_key_field='id',
+        outputs=accounts,
+        readable_output=tableToMarkdown(
+            'Azure Storage Account List',
+            readable_output,
+            ['Account Name', 'Subscription ID', 'Resource Group', 'Kind', 'Sku', 'Status Primary', 'Status Secondary', 'Location', 'Tags'],
+        ),
+        raw_response=response
+    )
+
+
+def storage_account_create_update(client: AKSClient, args: Dict) -> str:
+    account_args = {
+        'account_name': args.get('account_name'),
+        'sku': args.get('sku'),
+        'kind': args.get('kind'),
+        'location': args.get('location')
+    }
+    if args.get('tags'):
+        account_args['tags'] = args.get('tags')
+    if args.get('custom_domain_name'):
+        account_args['custom_domain_name'] = args.get('custom_domain_name')
+        if args.get('use_sub_domain_name'):
+            account_args['use_sub_domain_name'] = argToBoolean(args.get('use_sub_domain_name'))
+        else:
+            account_args['use_sub_domain_name'] = False
+    if args.get('enc_key_source'):
```

Comment:
use the walrus operator in all these conditions

Reply:
done.